packages/paste-core/components/side-modal/__tests__/customization.spec.tsx

```diff
@@ -116,7 +116,7 @@ describe('Customization', () => {
     );
 
     const showButton = screen.getByRole('button', {name: 'Button'});
-    await waitFor(() => {
+    await act(async () => {
```

Comment:
Praise: OMG that was the one thing I didn't try when I looked into this. Thank you

Reply:
haha yeah took me a few mins of guessing to figure it out myself